### PR TITLE
Refactor/client/types

### PR DESCRIPTION
**Pull request by nicdard**
Just a minor refactor of the client code as in: https://github.com/flix/vscode-flix/pull/177

Related to: #149

### CODE REVIEW

File: client/src/services/releases.ts [hunk 1]
```diff
@@ -69,19 +68,19 @@ export async function fetchRelease (
   // We skip runtime type checks for simplicity (here we cast from `any` to `GithubRelease`)
   const release: GithubRelease = await response.json()
   const flixRelease: FlixRelease = {
-    url: _.get('url', release),
-    id: _.get('id', release),
-    name: _.get('name', release),
-    version: tagToVersion(_.get('tag_name', release)),
-    downloadUrl: _.get('assets.0.browser_download_url', release),
+    url: _.get(release, 'url'),
```
Comment (magnus-madsen) [inline client/src/services/releases.ts:71]: Did the argument order change?

Comment (nicdard) [reply client/src/services/releases.ts:71]: Yes, as lodash get type definition, the function works well (I have tested it)

Comment (nicdard) [reply client/src/services/releases.ts:71]: it's somewhat strange I agree, but the type definition differs from what was written here

Comment (magnus-madsen) [reply client/src/services/releases.ts:71]: Surprising!

**Comment by magnus-madsen** [comment]
Thanks!

**Pull request merged**